.goreleaser.yml

```diff
@@ -131,3 +131,54 @@ winget:
           owner: go-task
           name: winget-pkgs
           branch: "bump-task-to-{{.Tag}}"
+
+dockers:
+  - image_templates:
+      - "ghcr.io/go-task/task:v{{.Version}}-amd64"
+    use: buildx
+    dockerfile: Dockerfile
+    build_flag_templates:
+      - "--platform=linux/amd64"
+      - "--label=org.opencontainers.image.created={{.Date}}"
+      - "--label=org.opencontainers.image.title={{.ProjectName}}"
+      - "--label=org.opencontainers.image.revision={{.FullCommit}}"
+      - "--label=org.opencontainers.image.version={{.Version}}"
+      - "--label=org.opencontainers.image.url=https://taskfile.dev"
+      - "--label=org.opencontainers.image.source=https://github.com/go-task/task"
+      - "--label=org.opencontainers.image.licenses=MIT"
+      - "--label=org.opencontainers.image.authors=The Task authors <task@taskfile.dev>"
+      - "--label=org.opencontainers.image.description=Task is a task runner / build tool that aims to be simpler and easier to use than, for example, GNU Make."
+  - image_templates:
+      - "ghcr.io/go-task/task:v{{.Version}}-arm64v8"
```

Comment:
I wouldn't append the variant for arm64, there is only one, so it's practically nowhere used:
https://github.com/search?q=repo%3Acontainerd%2Fcontainerd+arm64v8&type=code
When platforms are built separately then the common suffixes are `amd64` and `arm64`.